packages/flame/lib/src/cache/memory_cache.dart

```diff
@@ -19,6 +19,10 @@ class MemoryCache<K, V> {
     }
   }
 
+  void removeValue(K key) {
```

Comment:
In the `AssetsCache` this method is called `clear`, we should probably follow the same naming convention here.

Reply:
Added `clearCache()` function to clear all the caches.